apps/condo/domains/user/schema/SignInUserService.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Generated by `createservice user.SignInUserService --type mutations`
+ */
+
+const get = require('lodash/get')
+const isNil = require('lodash/isNil')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, getSchemaCtx, getById, getByCondition } = require('@open-condo/keystone/schema')
+
+const { NOT_FOUND, COMMON_ERRORS } = require('@condo/domains/common/constants/errors')
+const { normalizeEmail } = require('@condo/domains/common/utils/mail')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/user/access/SignInUserService')
+const { STAFF, RESIDENT } = require('@condo/domains/user/constants/common')
+const { GQL_ERRORS: USER_ERRORS } = require('@condo/domains/user/constants/errors')
+const { ConfirmPhoneAction, User } = require('@condo/domains/user/utils/serverSchema')
+const { checkDailyRequestLimitCountersByIp } = require('@condo/domains/user/utils/serverSchema/requestLimitHelpers')
+
+
+const REQUIRED_USER_FIELDS_BY_TYPE = {
+    [RESIDENT]: ['phone'],
+    [STAFF]: ['phone', 'name', 'password'],
+}
+const SUPPORTED_USER_TYPES = [RESIDENT, STAFF]
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    REQUIRED_USER_DATA_IS_MISSING: (missingFields = []) => ({
```

Comment:
it's better to leave it as static dict, and override it dynamically inside the validation

Reply:
Fixed